tmva/tmva/src/Classification.cxx

```diff
@@ -262,7 +262,7 @@ void TMVA::Experimental::Classification::Evaluate()
 #ifndef _MSC_VER
       fWorkers.SetNWorkers(fJobs);
 #endif
-      auto executor = [=](UInt_t workerID) -> ClassificationResult {
+      auto executor = [this](UInt_t workerID) -> ClassificationResult {
```

Comment:
I understand the issue, it is explained [here](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0806r2.html). 

I am not sure if the lambda uses local variables, to be sure the functionality is the 
same, should not be `[=,this]` ?

Reply:
It is compiled - means not used.